values.yaml

```diff
@@ -1,402 +1,144 @@
-environments:
-  # environments.tolerations -- Tolerations are applied to all user
-  # environments. Each element is a regular pod toleration object. To set
-  # service tolerations see serviceTolerations. See values.yaml for an example.
-  #
-  # e.g.
-  # tolerations:
-  # - key: "my key"
-  #   operator: "Exists"
-  #   effect: "NoExecute"
-  #   tolerationSeconds: 6000
-  tolerations: []
-
-  # environments.nodeSelector -- nodeSelector is applied to all user
-  # environments to specify eligible nodes for environments to run on.
-  # See: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector
-  # 
-  # eg.
-  # nodeSelector:
-  #   disktype: ssd
-  nodeSelector: {}
-
-# certs -- Describes CAs that should be added to Coder services. These certs
-# are NOT added to environments.
-certs:
-  secret:
-    # certs.secret.name -- The name of the secret.
-    name: ""
-    # certs.secret.key -- The key in the secret pointing to the certificate
-    # bundle.
-    key: ""
-
-# namespaceWhitelist -- DEPRECATED: A list of additional namespaces to which
-# environments may be deployed. Use multiple workspace providers instead.
-# Existing environments within these namespaces will continue to function, but
-# new environments cannot be created within these namespaces.
-#
-# If you used this value before 1.17, you must keep it set to the same value
-# until all environments within these namespaces are deleted, then you can set
-# it back to [].
-namespaceWhitelist: []
-
-ssh:
-  # ssh.enable -- Enables accessing environments via SSH.
-  enable: true
-
-# serviceTolerations -- Tolerations are applied to all Coder managed services.
-# Each element is a toleration object. To set user environment tolerations see
-# environments.tolerations. See values.yaml for an example.
-#
-# e.g.
-# serviceTolerations:
-# - key: "my key"
-#   operator: "Exists"
-#   effect: "NoExecute"
-#   tolerationSeconds: 6000
-serviceTolerations: []
-
-# Services contains global configuration that applies to all Coder managed
-# services.
+  # postgres.default -- Configure a built-in PostgreSQL deployment.
+  default:
+    # postgres.default.enable -- Deploys a PostgreSQL instance. We recommend
+    # using an external PostgreSQL instance in production.
+    # If true, all other values are ignored.
+    enable: true
+    # postgres.default.image -- Injected by Coder during release.
+    image: ""
+    # postgres.default.resources -- Kubernetes resource specification for the PostgreSQL pod.
+    # To unset a value, set it to "". To unset all values, set resources to nil.
+    resources:
+      requests:
+        cpu: "250m"
+        memory: "1Gi"
+        # postgres.default.resources.requests.storage -- Specifies the size of the volume claim
+        # for persisting the database.
+        storage: "10Gi"
+      limits:
+        cpu: "250m"
+        memory: "1Gi"
+
+# services -- Kubernetes Service configuration that applies to Coder services. 
```

Comment:
Can we roll this into coderd now that it's the only service we run?

Reply:
I was thinking that too, but we still optionally run PostgreSQL.

Reply:
ah yeah makes sense to apply to that too